### DISCUSSION TITLE
`use of undeclared crate or module pyo3` in windows 11

### DISCUSSION THREAD

**Discussion by sun-rs** [PyO3/pyo3#4336] [Questions]
I use pyo3 and set Cargo.toml as `pyo3 = { version = "0.21.2", features = ["hashbrown", "chrono", "anyhow", "abi3-py310"] }`, it works well in linux and macos, but the same code not work in win11, all pyo3/chrono/rayon became `failed to resolve: use of undeclared crate or module pyo3 use of undeclared crate or module pyo3`.
<img width="779" alt="图片" src="https://github.com/PyO3/pyo3/assets/8978871/e2986a65-81a6-4f37-a1dc-702e47d73a50">

Why is that? all the crates are correctly imported in Cargo.toml

**Comment by davidhewitt** [comment]
The problem isn't obvious to me nor does it sound PyO3 specific, you might have better luck asking for help using Rust on Windows in a general Rust help forum such as https://users.rust-lang.org
